test/test_nestedtensor.py

```diff
@@ -762,6 +748,20 @@ def test_nested_tensor_indexing(self, device, dtype):
         expected_grad = torch.nested.nested_tensor([grad_x0, torch.zeros((3, 4), device=device, dtype=dtype)])
         self.assertEqual(nt.grad, expected_grad)
 
+    @parametrize("func", [torch.nn.functional.relu,
+                          torch.nn.functional.relu_,
+                          torch.nn.functional.gelu,
+                          torch._C._nn.gelu_,
+                          torch.tanh,
+                          torch.tanh_,
+                          torch.neg])
+    def test_activations(self, device, func):
+        nt = random_nt(device, torch.float32, 3, (4, 4))
```

Comment:
```suggestion
        random_nt_noncontiguous_pair
```
Should we test both contiguous and noncontiguous with this func?

Reply:
Unary ops currently only support contiguous inputs.

Reply:
I'll add an assertRaises